src/main/java/uk/gov/justice/hmpps/prison/api/model/RollCount.java

```diff
@@ -117,6 +125,16 @@ public void setCurrentlyInCell(final Integer currentlyInCell) {
         this.currentlyInCell = currentlyInCell;
     }
 
+    @Schema(requiredMode = REQUIRED, description = "No of residential prisoners in internal locations")
```

Comment:
is this description correct - doesn't seem to correspond to the field name

Reply:
yep, its just another way of putting it.